drivers/gpu/drm/lima/lima_device.c

```diff
@@ -106,27 +106,46 @@ static int lima_gp_group_init(struct lima_device *dev)
 	if (!gp)
 		return -ENOMEM;
 
+	/* Init GP-group L2 cache on Mali450 */
+	if (dev->gpu_type == GPU_MALI450) {
+		gp->l2_cache = kzalloc(sizeof(*gp->l2_cache), GFP_KERNEL);
+		if (!gp->l2_cache) {
+			err = -ENOMEM;
+			goto err_out0;
+		}
+		gp->l2_cache->ip.irq = -1;
+		if ((err = lima_init_ip(dev, "l2-cache", &gp->l2_cache->ip, LIMA450_GPL2_BASE)) ||
```

Comment:
I'd prefer name gp-l2-cache to distinguish between pp l2 cache and mali400 l2-cache

Reply:
I've amended the commit to include the requested name changes.